examples/command_error.py

```diff
@@ -0,0 +1,61 @@
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.message_content = True
+
+positive_emoji: str = "\U00002705"
+negative_emoji: str = "\U0000274e"
+
+bot = commands.Bot(
+    command_prefix=commands.when_mentioned_or("!"),
+    intents=intents
+)
+
+
+# return True if channel was thread.
+def is_thread(ctx: commands.Context) -> bool:
```

Comment:
I feel like it would be a little better if the `is_thread` function check was defined right before the command that uses it